src/SlideShow/SlideShowNavigation.tsx

```diff
@@ -46,6 +50,8 @@ interface SlideShowNavigationProps {
   size: number
   active: number
   hideNavigationDots: boolean
+  darkMode: boolean
```

Comment:
You should not be needing this prop. Dark mode is already supported globally, using a theme.

Reply:
@alexisfontaine So it won't be handled locally in the `NavigationDots` component?

Reply:
Nope.